app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -42,7 +42,7 @@
 import com.google.common.collect.Maps;
 
 public class DynamoSurveyDao implements SurveyDao {
```

Comment:
There are still uncached code paths, and it only takes 2 r/s to potentially hit DDB throttling. Should we add a caching layer to the survey elements? Or is this unlikely enough that we don't need to defend against it?